avocado/core/output.py

```diff
@@ -639,6 +639,8 @@ def _log_line(self, line):
         Passes lines of output to the logging module.
         """
         for lg in self._logger:
+            if not lg.handlers:
+                lg.handlers.append(NULL_HANDLER())
```

Comment:
oh I just noticed this is in the `log_line`. I thought it's in init. Well wouldn't it be better to simply do `continue` in such case?